portfolio/src/main/java/com/google/sps/util/JsonUtil.java

```diff
@@ -0,0 +1,14 @@
+package com.google.sps.util;
+
+import java.util.ArrayList;
+
+import com.google.gson.Gson;
+
+public class JsonUtil {
+
+  private static final Gson GSON = new Gson();
```

Comment:
rename this to gson, as it is not a constant

Reply:
Done